pkg/grub/rootfs.cfg

```diff
@@ -308,7 +302,10 @@ function set_kvm_boot {
    set_global load_hv_cmd echo
    set_global load_dom0_cmd linux
    set_global load_initrd_cmd initrd
-   set_global dom0_extra_args "$dom0_extra_args pcie_acs_override=downstream,multifunction"
+   set_global dom0_flavor_tweaks "pcie_acs_override=downstream,multifunction"
+   if [ "$arch" = "x86_64" ]; then
+      set_global dom0_flavor_tweaks "$dom0_flavor_tweaks crashkernel=2G-64G:128M,64G-1T:256M,1T-:512M"
+   fi
```

Comment:
Currently we also set crashkernel with xen, but here you only set it with kvm. Is that what we want?

Reply:
Ah - I missed that. @rouming do you think we should keep the separate function set_crashkernel()?